sql/search_derived/mobile_search_clients_last_seen_v1/query.sql

```diff
@@ -0,0 +1,135 @@
+WITH _derived_search_cols AS (
+  SELECT
+    udf.normalize_search_engine(engine) AS short_engine,
+    COALESCE(organic, 0) + COALESCE(search_count, 0) + COALESCE(unknown, 0) + COALESCE(
+      tagged_sap,
+      0
+    ) + COALESCE(tagged_follow_on, 0) AS total_searches,
+    COALESCE(tagged_sap, 0) + COALESCE(tagged_follow_on, 0) AS tagged_searches,
+    COALESCE(ad_click, 0) AS ad_click,
+    COALESCE(search_with_ads, 0) AS search_with_ads,
+    * EXCEPT (ad_click, search_with_ads)
+  FROM
+    mobile_search_clients_daily_v1
+  WHERE
+    submission_date = @submission_date
+),
+_derived_engine_searches AS (
+  -- From the clients search info, make a struct
+  -- that we will use for aggregation later
+  SELECT
+    STRUCT(
+      short_engine AS key,
+      STRUCT(total_searches, tagged_searches, ad_click, search_with_ads) AS value
+    ) AS engine_searches,
+    * EXCEPT (short_engine)
+  FROM
+    _derived_search_cols
+),
+_grouped AS (
+  -- Here we get a single row per-client, containing
+  -- info from each engine, as well as overall info
+  SELECT
+    client_id,
+    ANY_VALUE(sample_id) AS sample_id,
```

Comment:
I'd prefer to put `sample_id` in the GROUP BY so that we can avoid using `ANY_VALUE`

```suggestion
    sample_id,
```